aea/cli/common.py

```diff
@@ -39,7 +40,7 @@
 logger = logging.getLogger("aea")
 logger = default_logging_config(logger)
 
-DEFAULT_REGISTRY_PATH = "../packages"
+DEFAULT_REGISTRY_PATH = str(Path(aea.AEA_DIR, "..", "packages").resolve())
```

Comment:
Are we sure this is sensible? For instance, in line 70 of aea/cli/create.py we now save the entire path to the config..